src/components/riveMonster/rive-monster-component.ts

```diff
@@ -54,16 +58,39 @@ export class RiveMonsterComponent {
   };
 
   private initializeHitbox() {
-    const scale = gameSettingsService.getDevicePixelRatioValue();
-    const monsterCenterX = (this.props.canvas.width / scale) / 2;
-    const monsterCenterY = (this.props.canvas.height / scale) / 2;
+    const monsterCenterX = (this.props.canvas.width / this.scale) / 2;
+    const monsterCenterY = (this.props.canvas.height / this.scale) / 2;
     const rangeFactorX = 55;
     const rangeFactorY = 100;
 
     this.hitboxRangeX = { from: monsterCenterX - rangeFactorX, to: monsterCenterX + rangeFactorX };
     this.hitboxRangeY = { from: monsterCenterY + (rangeFactorY / 2), to: monsterCenterY + (rangeFactorY * 2) };
   }
 
+  private getRiveMinYAdjustment() {
```

Comment:
We can also add a one line comment on why we are doing this minY adjustment @bernhardccodev